extension/keysocket-deezer.js

```diff
@@ -1,38 +1,23 @@
 function onKeyPress(key) {
-    if (document.getElementById('player_control_play')) { // old deezer style
+	if(document.querySelector('#player_control_play')) { // old deezer style
 		if(key === NEXT) {
-			var nextButton = document.getElementById('player_control_next');
-			simulateClick(nextButton);
+			simulateClick(document.querySelector('#player_control_next'));
 		} else if(key === PLAY) {
-			var isPlaying = document.getElementById('player_control_play').style.display === 'none';
-			var playPauseButton = null;
-			if(isPlaying) {
-			  playPauseButton = document.getElementById('player_control_pause');
-			} else {
-			  playPauseButton = document.getElementById('player_control_play');
-			}
-			simulateClick(playPauseButton);
+			simulateClick(document.querySelector('#player_control_play').style.display === 'none' ? document.querySelector('#player_control_pause') : document.querySelector('#player_control_play'));
```

Comment:
Using inline if prevents storing variables, would this be an acceptable alternative?
```
      simulateClick(
        document.querySelector('#player_control_play').style.display === 'none'
      ?
        document.querySelector('#player_control_pause')
      :
        document.querySelector('#player_control_play')
      );
```